sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/subquery.scala

```diff
@@ -587,6 +589,20 @@ object RewriteCorrelatedScalarSubquery extends Rule[LogicalPlan] {
           }
         }
     }
+    (newChild, AttributeMap(subqueryAttrMapping))
```

Comment:
`AttributeMap(subqueryAttrMapping.toSeq)` to pass scala 2.13 compilation

Reply:
Ah, I missed that. Updated.